libs/category/src/drivers/magento/category.service.ts

```diff
@@ -51,8 +55,22 @@ export class DaffMagentoCategoryService implements DaffCategoryServiceInterface
 			})
     ]).pipe(
       map((result): MagentoCompleteCategoryResponse => this.buildCompleteCategoryResponse(result[0].data, result[1].data, result[2].data)),
-      map((result: MagentoCompleteCategoryResponse) => this.magentoCategoryResponseTransformer.transform(result))
-    );
+			switchMap((categoryResult: MagentoCompleteCategoryResponse) => {
```

Comment:
Can this be done on the internal pipe to get the aggregations for the request as opposed to after all the other requests have completed?

Reply:
yes, done